src/pages/Profile.jsx

```diff
@@ -178,8 +181,8 @@ class _Profile extends Component {
             />
           </div>
 
-          <div className="profile__form-item profile__col">
-            <label className="profile__label" htmlFor="dob">University / School</label>
+          <div className={`profile__form-item profile__col ${this.state.me.school ? "" : "profile__form-item--error"}`}>
+            <label className="profile__label" htmlFor="dob">University / School *</label>
```

Comment:
You don't need to add JS complexity for showing a red border around an input when it is not valid.

Use validation attributes on the input like `required` and `minlength`.

To show a red border around an input use the invalidity css selector `input:invalid`.

Reply:
interesting! good point, I will fix that.